src/AddTodoForm.js

```diff
@@ -1,11 +1,26 @@
 import React from 'react';
 
-const AddTodoForm = () => (
-        <form>
+function AddTodoForm(props){
+
+    const handleAddTodo = (event) => {
+
+        event.preventDefault();
+
+        const todoTitle = document.querySelector('input[name=title]').value;
```

Comment:
This works, but the lesson says:

"retrieve the value of the title element from the event target".

You can do this is a few ways:

`event.target[0].value` - by index
`event.target.elements.title.value)` - using elements
`event.target.title.value` - or directly